outputs.tf

```diff
@@ -0,0 +1,67 @@
+################################################################################
+# mongodbatlas_project
+################################################################################
+
+output "project_id" {
+  value = mongodbatlas_project.project.id
+}
+
+output "creation_timestamp" {
+  value = mongodbatlas_project.project.created
+}
+
+################################################################################
+# mongodbatlas_project_ip_access_list
+################################################################################
+
+output "ip_ids" {
+  value = [for i in mongodbatlas_project_ip_access_list.ip :
+    {
+      ip_id      = i.id
+      ip_address = i.ip_address
+    }
+  ]
+}
+
+output "cidr_ids" {
+  value = [for c in mongodbatlas_project_ip_access_list.cidr :
+    {
+      cidr_id    = c.id
+      cidr_block = c.cidr_block
+    }
+  ]
+}
+
+################################################################################
+# mongodbatlas_database_user
+################################################################################
+
+output "database_users" {
+  value = [for dbu in mongodbatlas_database_user.dbuser :
+    {
+      database_user = dbu.username
```

Comment:
why not `username = dbu.username`?

Reply:
wanted to give a more explanatory name, but if username is enough I can change it